tests/cli/test_datasource_pandas.py

```diff
@@ -53,11 +63,21 @@ def test_cli_datasorce_list(caplog, empty_data_context, filesystem_csv_2):
     result = runner.invoke(
         cli, ["datasource", "list", "-d", project_root_dir], catch_exceptions=False
     )
-
+    expected_output = """
+1 Datasource found:[0m
```

Comment:
for my learning: I'm not familiar with this syntax...what does it mean?

Reply:
oh, those are the ANSI color codes. I think in the future we can strip those out for tests via click runner. Side note, this is how windows users see GE.

Reply:
Not sure exactly - I believe they're color/other formatting things.

Reply:
👍 Thank you!